bodhi-server/bodhi-server.spec

```
@@ -125,7 +101,7 @@ install apache/bodhi.wsgi %{buildroot}%{_datadir}/bodhi/bodhi.wsgi
 install -d %{buildroot}%{_mandir}/man1
 install -pm0644 docs/_build/*.1 %{buildroot}%{_mandir}/man1/
 
-%files -n %{pypi_name}
+%files -n %{pypi_name} -f %{pyproject_files}
```

When you wish to put some files aside, don't use `%{pyproject_files}`. It is a shortcut that can be used only if all files should end up in the same RPM package. 

```suggestion
%files -n %{pypi_name}
```

---

Mea culpa, I've learned in https://bugzilla.redhat.com/show_bug.cgi?id=2049301 that you can actually keep using `%{pyproject_files}` if you add the `%exclude` lines to the package.

---

tl;dr this should be possible:

```
%files -n %{pypi_name} -f %{pyproject_files}
...
%pycached %exclude %{python3_sitelib}/bodhi/server/tasks/composer.py
%pycached %exclude %{python3_sitelib}/bodhi/server/tasks/metadata.py


%files -n bodhi-composer
...
%pycached %{python3_sitelib}/bodhi/server/tasks/composer.py
%pycached %{python3_sitelib}/bodhi/server/tasks/metadata.py
```